document-models/budget-statement/src/reducers/line-item.ts

```diff
@@ -21,8 +21,8 @@ function isEqual(
     lineItem: LineItem
 ) {
     return (
-        lineItemInput.category === lineItem.category?.id &&
-        lineItemInput.group === lineItem.group?.id
+        lineItemInput.category === lineItem.category &&
```

Comment:
I think this was a bug — the values we use in the tests never reference an `id` and treats the value as being the string itself. I'll add another comment by the test code I'm referring to.

Reply:
From what I remember `group` and `category` can either be an Id or a an object with `{id, name, ...}`.
If we need to actually use budget statements in the future we'll worry about it then.